edi_storage_oca/tests/test_edi_backend_storage.py

```diff
@@ -39,7 +39,7 @@ def test_export_file_already_done(self):
         self.record.edi_exchange_state = "output_sent"
         mocked_paths = {self._file_fullpath("done"): self.fakepath}
         # TODO: test send only w/out cron (make sure check works)
-        self._test_run_cron(mocked_paths)
+        self._test_run_cron(mocked_paths, skip_sent=False)
```

Comment:
why do we have to add this change?

Reply:
Because in `tests` that have some case with the status `output_sent` so we use the flag `skip_sent` to ignore another case:
- `skip_sent` = True --> ignore records that were already sent
- `skip_sent` = False ---> send it

Reply:
I know what it does :wink: 
I mean: why was not needed before and it is?